src/Microsoft.Extensions.ML/Microsoft.Extensions.ML.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.Extensions.ML</IncludeInPackage>
+    <NoWarn>$(NoWarn);NU5118</NoWarn>
```

Comment:
Can this property be included in some top level Directory.Build.props?

Reply:
Yes, that is great idea, I think we can.

---
In reply to: [538041772](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538041772) [](ancestors = 538041772)